app/models/miq_template/operations.rb

```diff
@@ -3,28 +3,16 @@ module MiqTemplate::Operations
 
   included do
     supports :clone
-  end
-
-  def validate_collect_running_processes
-    validate_invalid_for_template(_("VM Process collection"))
-  end
-
-  def validate_stop
-    validate_invalid_for_template(_("Stop Operation"))
-  end
-
-  def validate_pause
-    validate_invalid_for_template(_("Pause Operation"))
-  end
-
-  def validate_standby_guest
-    validate_invalid_for_template(_("Standby Guest Operation"))
-  end
-
-  private
-
-  def validate_invalid_for_template(message_prefix)
-    {:available => false,
-     :message   => _("%{message} is not available for Images/Templates.") % {:message => message_prefix}}
+    supports_not :collect_running_processes, :reason => _("Process Collection is not available for Images/Templates.")
```

Comment:
I think all of these reasons need to be `N_` as opposed to `_`

Reply:
...or they need to use a block.  I'm not sure if class-level `N_` are later going to looked up with `_` or not

Reply:
if these are `N_`, then we need to ensure that the callers in the UI do `_` after.  Is that correct?

Reply:
Yes I believe so

Reply:
@Fryguy why do you think this needs to be `N_()` as opposed to `_()`?